run_dir/design/flowcells.html

```diff
@@ -34,7 +34,7 @@
             <div class="tab-pane fade show active" id="tab_flowcells">
 
                 <div id="fc_table_div">
-                    <h1><span id="page_title">Illumina flowcells</span>
+                    <h1><span id="page_title">Flowcells</span>
```

Comment:
Is the change only for Illumina?
https://github.com/SciLifeLab/genomics-status/blob/07252c63593c52b27ba7c4071dc579a3561df963/run_dir/design/ont_flowcells.html#L4

Reply:
[c00a11a](https://github.com/SciLifeLab/genomics-status/pull/753/commits/c00a11ab597844e24d38bbdc3bf7b46de635e066) keeps the headers and only changes the web page title shown in the browser.